custom_components/tuya_local/devices/neatsvor_x520ru_vacuum.yaml

```diff
@@ -0,0 +1,153 @@
+name: neatsvor_x520_vacuum_cleaner
+products:
+  - id: dbougox8k7ntyxeh
+    name: Neatsvor X520RU
+    manufacturer: Neatsvor
+    model: X520RU
+entities:
+  - entity: vacuum
+    dps:
+      - id: 1
+        type: boolean
+        name: power
+      - id: 2
+        type: boolean
+        name: activate
+      - id: 3
+        type: string
+        name: command
+        mapping:
+          - dps_val: "standby"
+            value: "standby"
+          - dps_val: "smart"
+            value: "smart"
+          - dps_val: "wall_follow"
+            value: "wall_follow"
+          - dps_val: "mop"
+            value: "mop"
+          - dps_val: "spiral"
+            value: "clean_spot"
+          - dps_val: "chargego"
+            value: "return_to_base"
+      - id: 5
+        type: string
+        name: status
+        mapping:
+          - dps_val: "0"
+            value: "Standby"
+          - dps_val: "1"
+            value: "Smart cleaning"
+          - dps_val: "2"
+            value: "Mopping"
+          - dps_val: "3"
+            value: "Cleaning along the walls"
+          - dps_val: "4"
+            value: "Recharging"
+          - dps_val: "5"
+            value: "Charging"
```

Comment:
I don't know what the difference is supposed to be between "Recharging" and "Charging". Is one supposed to be "charged"?